src/test/java/org/mule/extension/db/integration/connectivity/DataSourcePoolingTestCase.java

```diff
@@ -67,6 +69,9 @@ public void connectionsGoBackToThePool() throws Exception {
 
   @Test
   public void limitsConnections() throws Exception {
+    //NOTE: ignore this test if it is run on an instance with less than 4 cores, otherwise it will fail.
+    Assume.assumeThat(Runtime.getRuntime().availableProcessors(), greaterThanOrEqualTo(4));
```

Comment:
Static import, `Assume.assumeThat()` -> `assumeThat()`.
Is part of the Mule Code Convention